js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -4,6 +4,8 @@ function leastCommonMultiple(number1, number2) {
 	if(typeof(number1) !== "number" || typeof(number2) !== "number") throw new Error(`Expected both numbers, Got ${typeof number1}, ${typeof number2}`)
 	let large = Math.max(number1, number2);
   let small = Math.min(number1, number2);
+  if(small == 0) return large
+
   let lcm = large;
   while(lcm % small !== 0) {
```

Comment:
The overall program is good, but I have a suggestion, that this program is working for only two parameters, you can also make it work for extra parameters if you want.

Reply:
yes. thanks for the feedback Gaurav.